ops/ansible/roles/bfd-db-migrator/test/test_basic.yml

```diff
@@ -0,0 +1,36 @@
+---
+##
+# Applies and verifies the role on the container being used for the test case.
+##
+
+# Apply the role.
+- hosts: bfd-db-migrator
+  tasks:
+    - name: Apply Role
+      import_role:
+        name: bfd-db-migrator
+      vars:
+        db_migrator_zip: "{{ lookup('env','HOME') }}/.m2/repository/gov/cms/bfd/bfd-db-migrator/1.0.0-SNAPSHOT/bfd-db-migrator-1.0.0-SNAPSHOT.zip"
```

Comment:
I assume at some point we will move to semver for this snapshot. Should this be using a wildcard for the version, or is this intended to be updated on each version increment?

Reply:
Someday™